bin/configs/java-restclient-useSingleRequestParameter.yaml

```diff
@@ -0,0 +1,9 @@
+generatorName: java
+outputDir: samples/client/petstore/java/restclient-useSingleRequestParameter
```

Comment:
can you please add the new output folder to https://github.com/OpenAPITools/openapi-generator/blob/master/.github/workflows/samples-java-client-jdk11.yaml so that the CI will test it moving forward?

Reply:
I added it to the jdk 17 action like the other Spring RestClient tests. (RestClient needs 17 as minimum) :)

Reply:
@wing328 Ping :) 🏓